documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -316,7 +316,7 @@ include::sds-developer-guide:ROOT:partial$sds-aspect-static-meta-model-java-arti
 
 ==== Providing Custom Macros for Code Generation
 
-It is possible to change predefined sections of the generated classes by providing custom https://velocity.apache.org/[Velocity] templates; see the https://velocity.apache.org/engine/1.7/user-guide.html[Velocity User Guide] for more information.
+It is possible to change predefined sections of the generated classes by providing custom https://velocity.apache.org/[Velocity] templates; see the https://velocity.apache.org/engine/2.3/user-guide.html[Velocity User Guide] for more information.
```

Comment:
We do not need to do this as part of this PR, but what do you think about externalizing the velocity version as parameter to the Antora build, at some point?

Reply:
Seems like a good idea to keep this up to date and not having a couple of positions to keep in mind after a version update.